packages/@n8n/task-runner/src/config/base-runner-config.ts

```diff
@@ -2,20 +2,20 @@ import { Config, Env, Nested } from '@n8n/config';
 
 @Config
 class HealthcheckServerConfig {
-	@Env('N8N_RUNNERS_SERVER_ENABLED')
+	@Env('N8N_RUNNERS_HEALTH_CHECK_SERVER_ENABLED')
```

Comment:
My thinking was to keep the env var generic so that, while right now it's only for health checks, if in future we want to use the server for something else as well, we wouldn't have to rename the env var. But if you think that's unlikely, then let's go ahead with this.

Reply:
I don't see a reason for the runner needing a server for anything else besides providing a health check endpoint, so I'd keep it explicit